Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -429,12 +429,27 @@ private void NetworkManager_NetworkChanged(object sender, EventArgs e)
             _validAddressResults.Clear();
         }
 
-        private Version _version;
+        private Version _applicationVersion;
         /// <summary>
-        /// Gets the current application version
+        /// Gets the current application server version
         /// </summary>
-        /// <value>The application version.</value>
-        public Version ApplicationVersion => _version ?? (_version = typeof(ApplicationHost).Assembly.GetName().Version);
+        /// <value>The application server version.</value>
+        public Version ApplicationVersion => _applicationVersion ?? (_applicationVersion = typeof(ApplicationHost).Assembly.GetName().Version);
```

Comment:
Shouldn't have both `ApplicationVersion` and `ApplicationSemanticVersion`. The semver version should be the `ApplicationVersion`. This means interfaces need to be changed to `string` instead of `Version`.

Having both is confusing as to which one is the source of truth.

Reply:
Alright well, the Version object is actually used in places as-is. I will rename the other one to ApplicationVersionString or something similar to make it more clear. The source of truth is neither anyway. That would be the AssemblyVersion attribute.

Reply:
I don't see a good reason to keep the `Version` instance. It is not accurate and should be removed.

Reply:
Well I’ll write a neater response tomorrow morning when I have my IDE again, but the InstallationManager uses is as the object itself, so it would need its versioning logic reimplemented possibly to make such a change possible.

That should really be a separate PR, and it adds a bunch of text parsing if there is version logic.

Reply:
This PR is replacing the exiting versioning scheme with our own. I'd like it to be all in this one unless you have a good reason.

Reply:
Nvm - I'll take care of it.